.claude/skills/bt-review/SKILL.md

```
@@ -17,34 +17,34 @@ Review current changes against project rules and quality standards.
 
 ## Steps
 
-1. **Gather changes**
+1. Gather changes
 
 - Run `git diff` to see all unstaged modifications
 - Run `git diff --cached` to see staged changes
 - List which files were modified and what changed
 
-2. **Run automated checks**
+2. Run automated checks
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)